rules/windows/process_creation/proc_creation_win_pua_adfind_susp_usage.yml

```diff
@@ -50,7 +62,7 @@ detection:
             - 'users_noexpire'
             - 'computers_active'
             - 'computers_pwdnotreqd'
-    condition: selection
+    condition: all of selection_*
```

Comment:
Same with this. Except if you have FPs (please provide if any). Rollback the logic.